src/mjolnir.py

```diff
@@ -57,11 +56,11 @@ def _admin_access_token(self) -> typing.Optional[str]:
         Returns:
             admin access token or None if fails.
         """
-        get_admin_access_token = getattr(self._charm, "get_admin_access_token", None)
-        if not get_admin_access_token:
+        access_token = self._charm_state.synapse_config.admin_access_token
+        if not access_token:
             logging.error("Failed to get method get_admin_access_token.")
```

Comment:
That error can be reworked now due to the change.

Reply:
Right now if the admin access token is None, the charm will be put on MaintenanceStatus. Do you think it should be changed?

self._charm.unit.status = ops.MaintenanceStatus(
                "Failed to get admin access token. Please, check the logs."
            )